src/nl/rubensten/texifyidea/window/EquationPreviewToolWindow.kt

```diff
@@ -0,0 +1,14 @@
+package nl.rubensten.texifyidea.window
+
+import javax.swing.*
+
+
+class EquationPreviewToolWindow(val equation_text:String) {
+    private val preview_form = PreviewForm()
```

Comment:
```suggestion

    private val preview_form = PreviewForm()
```
Insert newline above.